### ISSUE TITLE
Stable update breaks NetworkSerialPort on 24.09

### ISSUE THREAD

**Issue by a3f** [linux-automation/meta-lxatac#203]
I just updated to the newest bundle on the stable channel and the exported USB console is no longer functional and ser2net emits an error message (but still starts up):
```
Invalid accepter port name/number 'telnet(rfc2217,mode=server),48431': Invalid data to parameter
```

This is because the rootfs contains ser2net 4.6.1, which requires, for our purposes, that the command line contains `,tcp`. labgrid 24.0a2.dev32 installed on the target appears to not add this argument. The commit in question is @Emantor's https://github.com/labgrid-project/labgrid/commit/a3dbfb559 first released in [v24.0](https://github.com/labgrid-project/labgrid/releases/tag/v24.0).

System info:

```
root@lxatac-00018:~# cat /etc/buildinfo 
-----------------------
Build Configuration:  |
-----------------------
DISTRO = tacos
DISTRO_VERSION = 24.09
-----------------------
Layer Revisions:      |
-----------------------
meta              = HEAD:8637aa34f0009421eb6c51a93588fe879950e1f4
meta-poky         = HEAD:8637aa34f0009421eb6c51a93588fe879950e1f4
meta-ptx          = HEAD:547b079bf309ebe1576aa5ae0d58564feb245a42
meta-oe           = HEAD:1235dd4ed4a57e67683c045ad76b6a0f9e896b45
meta-python       = HEAD:1235dd4ed4a57e67683c045ad76b6a0f9e896b45
meta-perl         = HEAD:1235dd4ed4a57e67683c045ad76b6a0f9e896b45
meta-filesystems  = HEAD:1235dd4ed4a57e67683c045ad76b6a0f9e896b45
meta-networking   = HEAD:1235dd4ed4a57e67683c045ad76b6a0f9e896b45
meta-lxatac-bsp   = HEAD:6204945f502834569d769daf071cc83606897dce
meta-lxatac-software = HEAD:6204945f502834569d769daf071cc83606897dce
meta-labgrid      = HEAD:95308c3d2ac1b101481d3f3e4590a98f1fbeb2cb
meta-rauc         = HEAD:1e3e6b334defd7fbf95cb43d23975e7b3de4b520
meta-virtualization = HEAD:54b806b1985f3989722ee308e1073530fe3328c1
meta-selinux      = HEAD:9f5a46620a07d7b15722593a33e46a4d19392b75
```

**Comment by a3f** [comment]
It appears this is also broken in the testing channel.
Workaround:
```
sed -i.bak -r 's/telnet\(rfc2217,mode=server\)/telnet(rfc2217,mode=server),tcp/' /usr/lib/python3.*/site-packages/labgrid/remote/exporter.py
```

**Comment by hnez** [comment]
Hi, 

this should be fixed now with the release of version [v24.09.1](https://github.com/linux-automation/meta-lxatac/releases/tag/v24.09.1), which is now available on the [update server](https://downloads.linux-automation.com/lxatac/software/stable/).
Users of the stable channel will be prompted to update soon.

> It appears this is also broken in the testing channel

That is due to an unrelated reason, as this issue has already been fixed on the current development branch (scarthgap) by https://github.com/linux-automation/meta-lxatac/pull/197 (which has upgraded labgrid to a newer master version with grpc support).

The testing channel is still affected though because there has not been a successful run of [the testsuite](https://github.com/linux-automation/labgrid-lxatac/) that runs before a new testing upload is made. We are working on fixing these test failures so we see testing uploads again soon.
